src/main/scala/jigg/pipeline/Pipeline.scala

```diff
@@ -148,15 +150,13 @@ Currently the annotators listed below are installed. See the detail of each anno
         annotatorList.foldLeft(RequirementSet()) { (satisifedSofar, annotator) =>
           try annotator.checkRequirements(satisifedSofar)
           catch { case e: RequirementError =>
-            println("***** WARNING *****")
-            println(e.getMessage)
-            println("*******************")
+            System.out.println(s"[warn] %s".format(e.getMessage))
             satisifedSofar
           }
         }
       }
       case "false" => {
-        println("Skip `checkRequirements` for all annotators.")
+        System.out.println("[warn] SKIP `checkRequirements` for all selected annotators.")
```

Comment:
Outputting messages in standard output may cause errors since some users may connect the output of jigg in pipe.